Makefile

```diff
@@ -248,12 +248,16 @@ kernel: ## Build linux kernel.
 		--file neonvm/hack/Dockerfile.kernel-builder \
 		neonvm/hack
 
+.PHONY: check-local-context
+check-local-context: ## Asserts that the current kubectl context is pointing at k3d or kind, to avoid accidentally applying to prod
+	if [ "$$($(KUBECTL) config current-context)" != 'k3d-neonvm' ] && [ "$$($(KUBECTL) config current-context)" != 'kind-kind' ]; then echo "kubectl context is not pointing to local k3d or kind cluster (must be k3d-neonvm or kind-kind)"; fi
```

Comment:
Should we add exit with an error in case of unexpected context? Or does `make` do it automagically?

Reply:
Oh, I completely missed that :sweat_smile:

I don't think it does, no.